src/test/java/uk/gov/ons/ssdc/caseprocessor/service/UacServiceTest.java

```diff
@@ -29,7 +30,7 @@
   @InjectMocks UacService underTest;
 
   @Test
-  public void saveAndEmitUacUpdatedEvent() {
+  void saveAndEmitUacUpdatedEvent() {
```

Comment:
test method should remain public... should also be prefixed `test` but that's another story